.storybook/recipes/BaselineCard/BaselineCard.tsx

```diff
@@ -82,35 +82,31 @@ export const BaselineCard = ({
   const linkRef = useRef() as MutableRefObject<HTMLAnchorElement>;
 
   /**
-   * useEffect hook
```

Comment:
I removed some comments where they just reworded code that's pretty self-explanatory. (I wasn't consistent about this though because it's easier to just copy/paste than to analyze the comment and determine whether it should be removed or reworded, etc.)